databox-start

```diff
@@ -16,6 +16,12 @@ die() {
   err "$@"
   exit $rc
 }
+assert() {
+  if [ "$1" != "$2" ] 
```

Comment:
Google shell script style guide suggests `[[ ... ]]` is better
(https://google.github.io/styleguide/shell.xml)

Reply:
done